### ISSUE TITLE
KIWI_DONT_ENFORCE_HTTPS now working

### ISSUE THREAD

**Issue by DMyachin** [kiwitcms/Kiwi#1139]
### Steps to Reproduce

1. docker-compose down
2. vim /opt/Kiwi/docker-compose.yml
```
    web:
        container_name: kiwi_web
        depends_on:
            - db
        restart: always
        image: kiwitcms/kiwi:latest
        ports:
            - 80:8080
            - 443:8443
        volumes:
            - uploads:/Kiwi/uploads:Z
        environment:
            KIWI_DB_HOST: db
            KIWI_DB_PORT: 3306
            KIWI_DB_NAME: kiwi
            KIWI_DB_USER: kiwi
            KIWI_DB_PASSWORD: kiwi
            KIWI_DONT_ENFORCE_HTTPS: 'true'
```
3. docker-compose up -d
4. open http://ip/

### Actual results
Opened https://ip/

### Expected results
Opened https://ip/

And also KIWI_DONT_ENFORCE_HTTPS: true can not be applied:
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.web.environment.KIWI_DONT_ENFORCE_HTTPS contains true, which is an invalid type, it should be a string, number, or a null
```

**Comment by atodorov** [comment]
this is available in version 7.0 which will be released tomorrow.

are you testing a Docker image built from source or not? Which version of the image and of docker/docker-compose you are using ?

**Comment by atodorov** [comment]
No additional information provided in the past 10 days so closing. Reopen when you can retest with the latest version and provide more information if the issue still persists.
